### PR TITLE
feat: dataset details distribution accordion

### PR DESCRIPTION
**Pull request by inderps**
Screenshot 2024-02-26 at 15 30 57
Screenshot 2024-02-26 at 15 31 00

### CODE REVIEW

**Comment by inderps** [comment]
Separate PR for colorning suggestions

File: src/app/datasets/[id]/DistributionAccordion.tsx [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+              </div>
+              <div>
+                <strong className="block text-sm font-semibold">
+                  Compress Format:
+                </strong>
+                <span className="text-sm">{distribution.compressFormat}</span>
+              </div>
+              <div>
+                <strong className="block text-sm font-semibold">Issued:</strong>
+                <span className="text-sm">{distribution.issued}</span>
```
Comment (brunopacheco1) [inline src/app/datasets/[id]/DistributionAccordion.tsx:62]: European standard for date is `dd MMMM yyyy`.

**Comment by admy7** [comment]
Can you try with a higher border radius for the blue container please?

**Comment by brunopacheco1** [comment]
I thought you were trying to use some built-in components.

Anyway, the corners are too sharp, and a bit against the overal look&feel of other components, like the toolbar actions and buttons. What do you think of rounding them? Perhaps the same with the sidebar.

**Comment by inderps** [comment]
I was taking the flat design approach but i can change it to rounded if that makes it look more attractive

File: src/types/dataset.types.ts [hunk 2]
```diff
@@ -92,4 +92,43 @@ export interface Dataset {
   type: string;
   url: string;
   wasGeneratedBy: string;
+  distributions: DatasetDistribution[];
+}
+
+export interface DatasetDistribution {
```
[the remark this answers is not in the document]
Comment (inderps) [reply src/types/dataset.types.ts:98]: Yes there are some fields that are similar to datasets however we gain very little with deriving and it affects the code readbility

**Review by nolliia — APPROVED** [review]

File: src/app/datasets/[id]/DistributionAccordion.tsx [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+import React, { useState, useRef, useEffect } from "react";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import {
+  faChevronDown,
+  faChevronUp,
+  faFile,
+} from "@fortawesome/free-solid-svg-icons";
+import { DatasetDistribution } from "@/types/dataset.types";
+
+export default function DistributionAccordion({
```
Comment (admy7) [inline src/app/datasets/[id]/DistributionAccordion.tsx:14]: I have seen different ways of typing and exporting React components within this project. Maybe we should align to be consistent

**Review by admy7 — APPROVED** [review]

**Pull request merged**